src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -450,11 +302,206 @@ impl HummockVersionDeltaExt for HummockVersionDelta {
         self.level_deltas
             .values()
             .flat_map(|item| item.level_deltas.iter())
-            .map(|level_delta| level_delta.inserted_table_infos.len())
+            .map(|level_delta| match level_delta.get_delta_type().unwrap() {
```

Comment:
`get_sstableinfo_cnt` is not used at all, and its meaning is not clear. Can we remove it from HummockVersionDeltaExt trait?

Reply:
> `get_sstableinfo_cnt` is not used at all, and its meaning is not clear. Can we remove it from HummockVersionDeltaExt trait?

removed